TLM/TLM/Util/PriorityRoad.cs

```diff
@@ -238,16 +238,19 @@ public static class PriorityRoad {
 
         private static void HandleSplitAvenue(List<ushort> segmentList, ushort nodeId) {
             Log._Debug($"HandleSplitAvenue(segmentList, {nodeId}) was called");
+
+            // TODO: make static?
             void SetArrows(ushort segmentIdSrc, ushort segmentIdDst) {
                 LaneArrows arrow = ToLaneArrows(GetDirection(segmentIdSrc, segmentIdDst, nodeId));
-                ExtSegmentManager extSegmentManager = ExtSegmentManager.Instance;
-                IList<LanePos> lanes = extSegmentManager.GetSortedLanes(
-                                segmentIdSrc,
-                                ref segmentIdSrc.ToSegment(),
-                                extSegmentManager.IsStartNode(segmentIdSrc, nodeId),
-                                LaneArrowManager.LANE_TYPES,
-                                LaneArrowManager.VEHICLE_TYPES,
-                                true);
```

Comment:
This `true` would reverse the sort order but there didn't seem to be any need so I removed it. If it is required, then surely it should be `!LHT`?

Reply:
@kianzarrin is a sort order reversal required here?

Reply:
Makes sense.
test it with a Y junction anyway. an avenue splits into 2 highways. Angle matters.

Reply:
I've added the `true` back to retain existing approach; not sure if needed but outside scope of this PR to change functionality.